wordpress-mobile/WordPress-iOS
# Notifications: Fix detailed notification view not scrolling in landscape

Fixes #18811 

## Description
Fixes an issue where the detailed notification view was not scrollable in landscape mode.

## Testing Instructions
 
1. Complete a Next Steps tour and get a notification
2. Change phone to landscape mode
3. Open navigations tab
4. Click on the notification
5. Make sure that the view is scrollable

## Regression Notes
1. Potential unintended areas of impact
N/A
2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A
3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
